Foundation/FileHandle.swift

```diff
@@ -426,6 +434,17 @@ open class FileHandle : NSObject, NSSecureCoding {
     }
 #endif
 
+    internal convenience init?(fileSystemRepresentation: UnsafePointer<Int8>, flags: Int32, createMode: Int) {
+        let fd = _CFOpenFileWithMode(fileSystemRepresentation, flags, mode_t(createMode))
+        guard fd > 0 else { return nil }
+        
+#if os(Windows)
+        self.init(handle: HANDLE(bitPattern: _get_osfhandle(fd))!, closeOnDealloc: true)
```

Comment:
Why can't you use the `init(fileDescriptor:closeOnDealloc:)`?

Reply:
I did not notice that `init` method; I will update the code. Thank you.